### PR TITLE
Update Docs

### PR DESCRIPTION
**Pull request by rounak-adhikary**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,47 @@
...
+For documentation, please visit [Dell Terraform Providers documentation](https://dell.github.io/terraform-docs/).
+
+## Table of Contents
+
+* [Code of Conduct](./docs/CODE_OF_CONDUCT.md)
+* [Maintainer Guide](./docs/MAINTAINER_GUIDE.md)
+* [Committer Guide](./docs/COMMITTER_GUIDE.md)
+* [Contributing Guide](./docs/CONTRIBUTING.md)
+* [Branching Strategy](./docs/BRANCHING.md)
+* [List of Adopters](./docs/ADOPTERS.md)
```
Comment (gokul-srivathsan) [inline README.md:34]: Check ADOPTERS.md file

File: README.md [hunk 2]
```diff
@@ -1,2 +1,47 @@
...
+
+For documentation, please visit [Dell Terraform Providers documentation](https://dell.github.io/terraform-docs/).
+
+## Table of Contents
+
+* [Code of Conduct](./docs/CODE_OF_CONDUCT.md)
+* [Maintainer Guide](./docs/MAINTAINER_GUIDE.md)
+* [Committer Guide](./docs/COMMITTER_GUIDE.md)
+* [Contributing Guide](./docs/CONTRIBUTING.md)
+* [Branching Strategy](./docs/BRANCHING.md)
```
Comment (gokul-srivathsan) [inline README.md:33]: check CONTRIBUTING.md file

File: README.md (hunk 1 above)
Comment (rounak-adhikary) [reply README.md:34]: I think this is okay. We do not have any adopters yet in any case

File: README.md (hunk 2 above)
Comment (rounak-adhikary) [reply README.md:33]: checked.

**Review by Krunal-Thakkar — APPROVED** [review]

**Review by gokul-srivathsan — APPROVED** [review]
LGTM

**Review by Krunal-Thakkar — APPROVED** [review]

**Review by Krishnan-Priyanshu — APPROVED** [review]
LGTM

**Review by Krishnan-Priyanshu — APPROVED** [review]
LGTM

**Review by Sakshi-dell — APPROVED** [review]

**Review by AnikaAgiwal2711 — APPROVED** [review]

**Pull request merged**